### PR TITLE
[CWS] fix race with multiple client

### PR DESCRIPTION
**Pull request by safchain**
### What does this PR do?

It fix data race issues when having multiple ebpfless clients

### CODE REVIEW

File: pkg/security/probe/probe_ebpfless.go [hunk 1]
```diff
@@ -470,28 +480,40 @@ func (p *EBPFLessProbe) Start() error {
...
+	p.wg.Add(1)
 	go func() {
+		defer p.wg.Done()
+
 		for {
 			conn, err := listener.Accept()
 			if err != nil {
-				seclog.Errorf("unable to accept new connection")
-				continue
+				return
```
Comment (YoannGh) [inline pkg/security/probe/probe_ebpfless.go:490]: Should we keep logging errors that are unrelated with the listener being closed?

**Review by spikat — APPROVED** [review]

**Review by YoannGh — APPROVED** [review]
